alerts/apps_alerts.libsonnet

```diff
@@ -32,11 +32,11 @@
             // label exists for 2 values. This avoids "many-to-many matching
             // not allowed" errors when joining with kube_pod_status_phase.
             expr: |||
-              sum by (namespace, pod) (
-                max by(namespace, pod) (
+              sum by (namespace, pod, %(clusterGroupLabelsStr)s) (
```

Comment:
Could this be using [`clusterLabel`](https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/config.libsonnet#L84) ? I don't see why we would need another variable for this.

Same question for all cases with `clusterGroupLabelsStr`

Reply:
The default value for `clusterGroupLabels` is `clusterLabel`, so in the simple case they are the same thing.

But `clusterLabel` is used in other places (e.g. in dashboards as a selector like `%(clusterLabel)s="$cluster"`) and so can only be a single label.

Using a different variable for grouping labels gives more flexibility and allows use cases like I described in the PR, where I ideally want to group by `$clusterLabel, dc` and not just the `clusterLabel`.